packages/api/src/graphql/query.private.ts

```diff
@@ -189,6 +197,37 @@ export const GraphQLQuery = new GraphQLObjectType<undefined, Context>({
       }
     },
 
+    createJWTForUser: {
+      type: GraphQLJWTToken,
+      args: {
+        userId: {type: GraphQLNonNull(GraphQLString)},
+        expiresInMinutes: {type: GraphQLNonNull(GraphQLInt)}
+      },
+      async resolve(
+        root,
+        {userId, expiresInMinutes},
+        {authenticate, generateJWT, dbAdapter},
+        info
+      ) {
+        const THIRTY_DAYS_IN_MIN = 30 * 24 * 60
```

Comment:
Should go into settings. If not available yet, open a jira-story, so that we don't forget and can deliver it, as soon as the task is completed.

Reply:
https://wepublish.atlassian.net/browse/BAJ-548